compass/landice/tests/enthalpy_benchmark/A/namelist.landice3

```diff
@@ -1,4 +1,4 @@
 config_do_restart = .true.
-config_start_time = '150000-01-01_00:00:00'
+config_start_time = '150001-01-01_00:00:00'
```

Comment:
Is this change necessary?

Reply:
This is phase 3, so same answer as previous.